### PR TITLE
Add example to use event subscription without using side car

### PR DESCRIPTION
**Pull request by aneeshkp**
Shows examples of how to HTTP protocol directly with the producer.
This is not tested or supported yet

### CODE REVIEW

File: examples/simplehttp/common/httpclient.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+// Post ...
+func Post(address string, e cloudevents.Event) (int, error) {
+	sendCtx, sendCancel := context.WithTimeout(context.Background(), cancelTimeout)
+	defer sendCancel()
+	p, err := cloudevents.NewHTTP()
+	if err != nil {
+		log.Errorf("failed to create protocol: %s", err.Error())
+		return http.StatusBadRequest, err
+	}
+	c, err := cloudevents.NewClient(p, cloudevents.WithUUIDs(), cloudevents.WithTimeNow())
```
Comment (jzding) [inline examples/simplehttp/common/httpclient.go:28]: Is this c=NewClient object cleaned up somewhere?

File: examples/simplehttp/common/httpclient.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	if response.StatusCode == http.StatusOK {
+		bodyBytes, err = io.ReadAll(response.Body)
+		if err != nil {
+			return []byte(err.Error()), http.StatusBadRequest, err
+		}
+	} else {
+		bodyBytes, err = io.ReadAll(response.Body)
+		if err != nil {
+			return []byte(err.Error()), http.StatusBadRequest, err
+		}
```
Comment (jzding) [inline examples/simplehttp/common/httpclient.go:66]: Same actions are performed for if{} and else{}, then there is no need for if-else statement.

File: examples/simplehttp/main.go [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+func help() {
+	fmt.Println("--------------------------------")
+	fmt.Println("1. setting up port forwarding to the container port to expose the service external to k8s")
+	fmt.Println(" oc get pods -n openshift-ptp -l app=linuxptp-daemon")
+	fmt.Println("NAME                            READY   STATUS    RESTARTS   AGE")
+	fmt.Println("linuxptp-daemon-xmjcr           3/3     Running   0          13h")
+	fmt.Println("")
+	fmt.Println("Run following command to set port forwarding")
+	fmt.Println("oc -n openshift-ptp linuxptp-daemon-xmjcr 9043:9043 --address='0.0.0.0'")
```
Comment (jzding) [inline examples/simplehttp/main.go:161]: not a correct command

File: examples/simplehttp/common/httpclient.go (hunk 1 above)
Comment (aneeshkp) [reply examples/simplehttp/common/httpclient.go:28]: provided transport object and func scoped

File: examples/simplehttp/common/httpclient.go (hunk 2 above)
Comment (aneeshkp) [reply examples/simplehttp/common/httpclient.go:66]: good one

File: examples/simplehttp/external-service.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      targetPort: 3306
+      nodePort: 0
+---
+apiVersion: "v1"
+kind: "Endpoints"
+metadata:
+  name: "event-consumer-external"
+subsets:
+  - addresses:
+      - ip: "10.22.32.248"  #change this to your laptop ip address
```
[the remark this answers is not in the document]
Comment (aneeshkp) [reply examples/simplehttp/external-service.yaml:21]: 10.x is private. wont matter i guess

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**